.github/workflows/rm_gh_cache_after_pr_closed.yml

```diff
@@ -0,0 +1,45 @@
+name: Cleanup caches after PR closed
+on:
+  pull_request:
+    # branches:
+    #   - dev
```

Comment:
So this action can be triggered on PRs towards any branch, is that intended?

Reply:
Yes. Intended. Will remove.